pkg/local/configmaps_test.go

```diff
@@ -0,0 +1,136 @@
+// Copyright (c) 2020, Oracle and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package local
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"testing"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/stretchr/testify/assert"
+	v1beta1v8o "github.com/verrazzano/verrazzano-crd-generator/pkg/apis/verrazzano/v1beta1"
+	"github.com/verrazzano/verrazzano-operator/pkg/constants"
+)
+
+func TestNewConfigMap(t *testing.T) {
+	type args struct {
+		binding *v1beta1v8o.VerrazzanoBinding
+	}
+	tests := []struct {
+		name         string
+		args         args
+		expectedSize int
+		wantErr      bool
+	}{
+		{
+			name: "system",
+			args: args{
+				&v1beta1v8o.VerrazzanoBinding{
+					ObjectMeta: metav1.ObjectMeta{
+						Name: "system",
+					}},
+			},
+			expectedSize: 2,
+		}, {
+			name: "bookstore",
+			args: args{
+				&v1beta1v8o.VerrazzanoBinding{
+					ObjectMeta: metav1.ObjectMeta{
+						Name: "bookstore",
+					}},
+			},
+			expectedSize: 19,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			cm, err := newConfigMap(tt.args.binding)
+			if (err != nil) != tt.wantErr {
+				t.Errorf("newConfigMap() error = %v, wantErr %v", err, tt.wantErr)
+				return
+			}
+			assert.Equal(t, tt.expectedSize, len(cm.Data), "Expected size of Data")
+		})
+	}
+}
+
+type Dashboard struct {
+	Title  string  `json:"title"`
+	Panels []Panel `json:"panels"`
+}
+
+type Panel struct {
+	Title  string  `json:"title"`
+	Type   string  `json:"type"`
+	Id     int     `json:"id"`
+	Panels []Panel `json:"panels"`
+}
+
+func TestSystemDashboard(t *testing.T) {
```

Comment:
I would suggest splitting into 2 tests: TestSystemDashboardUniqueIds and TestSystemDashboardCPUPanel

Reply:
done